### ISSUE TITLE
A11y addon stopped working

### ISSUE THREAD

**Issue by Hypnosphi** [storybooks/storybook#4137]
http://storybooks-official.netlify.com/?selectedKind=UI%7CMenuItem&selectedStory=default&full=0&addons=1&stories=1&panelRight=0&addonPanel=%40storybook%2Faddon-a11y%2Fpanel

Expected: a warning about black text on blacky background
Actual: 0 violations 0 passes

**Comment by ndelangen** [comment]
Axe assumes default styling (white background on body).

however the background of the iframe is transparent, and below you see the color of the manager UI. (which is dark)

Axe cannot possibly know this, I think.

Is A11Y addon even enabled for that story?

**Comment by ndelangen** [comment]
I do not think this is a bug.

**Comment by ndelangen** [comment]
> why the tab is present

All tabs are always present, we do not have a system to only display tabs that are relevant.
We should add such a thing, sure.

> Should we maybe make iframe background white by default?

It's light on the light theme, it's dark in the dark theme. User have the ability to theme or add a preview-head.html or load css some other way to give the body inside the iframe a colour. Also they can use the background-addons.

The background addon works by adding the colour to the background of the preview (Component in the manager) So the background inside the iframe should be transparent.

I do not think we should force a white background by default.

> I think that the fact that our "official examples" site contains an unreadable text on the start screen is a major issue itself

Fixed it:
https://deploy-preview-4086--storybooks-official.netlify.com/?path=/components/Components-Heading-types

**Comment by Hypnosphi** [comment]
The transparent background also breaks full screen mode: http://storybooks-official.netlify.com/full=1

WDYT on this point?
> I think we should enable it for our own components and make sure that everything passes, so that we would be sure that storybook UI is accessible enough, at least with builtin themes.

**Comment by Hypnosphi** [comment]
> Fixed it:
https://deploy-preview-4086--storybooks-official.netlify.com/?path=/components/Components-Heading-types

=( https://deploy-preview-4086--storybooks-official.netlify.com/iframe.html?path=/components/Components-Heading-types

**Comment by ndelangen** [comment]
Fullscreen going white is a known issue I want to fix.

> I think we should enable it for our own components and make sure that everything passes, so that we would be sure that storybook UI is accessible enough, at least with built-in themes.
> WDYT on this point?

Hmm, I need to think about it.

**Comment by ndelangen** [comment]
Closing this as strictly speaking the addon works, I'll improve the official example in an attempt to make it as accessible as possible.
